ftva_lab_data/tests.py

```diff
@@ -704,3 +705,40 @@ def test_having_required_field_allows_save(self):
         new_item = form.save()
         self.assertIsNotNone(new_item)
         self.assertIsInstance(new_item, SheetImport)
+
+
+class DataExportTestCase(TestCase):
+    """Tests the format_data_for_export function."""
+
+    fixtures = ["sample_data.json", "item_statuses.json"]
+
+    def setUp(self):
+        # Create a test user to assign to a SheetImport object
+        self.user = get_user_model().objects.create_user(
+            username="testuser", first_name="Example", last_name="User"
+        )
+        SheetImport.objects.filter(pk=2).update(assigned_user=self.user)
+
+        # Give the other SheetImport object two statuses
+        SheetImport.objects.get(pk=3).status.add(1, 2)
+
+        # Create a list of SheetImport objects to test with
+        self.rows = [SheetImport.objects.get(pk=2), SheetImport.objects.get(pk=3)]
+
+    def test_format_data_for_export(self):
+        data_dicts = [row.__dict__ for row in self.rows]
+        export_data = format_data_for_export(data_dicts)
+
+        # Check that the carrier_a_location column is next to the carrier_a column
```

Comment:
With the carrier location fields moved in the model, that part of this test could be removed (though it does still pass... but by changing the model, testing position for carrier is not necessary).